### ISSUE TITLE
Problems with logout

### ISSUE THREAD

**Issue by justynrook** [opensource-socialnetwork/opensource-socialnetwork#476]
If I don't press the logout button I am unable to log back in.
Also if I am logged in somewhere else I am unable to login on another computer
Is there a timout? Should have something.

ossn 2.3

**Comment by lianglee** [comment]
Its seems the issue with your web browser, Try it on https://www.opensource-socialnetwork.org/demo/

**Comment by justynrook** [comment]
Hi there thanks for the reply.
I just tested yours and it works perfectly fine.
I am very confused now because I did a brand new install without any changed files and it was the same thing.
All my users are complaining they can't login or out.
this is what happens

1)put username and password in
2)Click sign in
3)Page just refreshes and username and password go blank.

When logging out, I have the same problem.
I hit logout and my news feed just refreshes.

HOWEVER,

If I try to login and it doesnt work, I can then type http://www.mysite.com/home   
if I click on that link after attempting to login it will take me to my feed.

So the session is starting but for some reason redirect is not working.

Where do I find the part that redirects to http://www.mysite.com/home   
and why wouldnt it be redirecting

thanks


Date: Mon, 3 Aug 2015 02:40:03 -0700
